rust/src/external/ntp.rs

```diff
@@ -0,0 +1,396 @@
+/* Copyright (C) 2017 Open Information Security Foundation
```

Comment:
Ok, will do in next PR
It's cleaner, especially if parsers use several files - the only drawback is that it tends to add more item to the 'use' statements.